nodes/node.go

```diff
@@ -10,19 +10,18 @@ import (
 	"github.com/stampzilla/gozwave/database"
 	"github.com/stampzilla/gozwave/events"
 	"github.com/stampzilla/gozwave/functions"
-	"github.com/stampzilla/gozwave/serialapi"
+	"github.com/stampzilla/gozwave/interfaces"
 )
 
-func New(address int, connection *serialapi.Connection, eventQue chan interface{}) *Node {
+func New(address int) *Node {
```

Comment:
In zwave protocol a device id always is a byte. thats why max size is 256 nodes. 
Are there other advantages using this:
```
type DeviceAddres byte
```
except from the code being easier to read since we know what is is by just looking at it like this?
```
func New(address DeviceAddres) *Node {
```